dbcon/mysql/ha_mcs_impl.cpp

```diff
@@ -2832,7 +2832,7 @@ void ha_mcs_impl_start_bulk_insert(ha_rows rows, TABLE* table, bool is_cache_ins
             (thd->lex)->sql_command == SQLCOM_INSERT_SELECT ||
             ci->isCacheInsert) && !ci->singleInsert )
     {
-        ci->useCpimport = get_use_import_for_batchinsert(thd);
```

Comment:
IMHO it is better to chage ci->useCpimport type than using the cast.

Reply:
I agree. Done.